language/move-prover/bytecode/src/dataflow_analysis.rs

```diff
@@ -190,15 +190,14 @@ pub trait DataflowAnalysis: TransferFunctions {
             },
         );
         while let Some(block_id) = work_list.pop_front() {
-            let pre = state_map.remove(&block_id).expect("basic block").pre;
-            let post = self.execute_block(block_id, pre.clone(), &instrs, cfg);
```

Comment:
I guess this clone was introduced later, and then indeed the original design with the remove does not make sense anylonger.